lib/Runtime/Library/MathLibrary.cpp

```diff
@@ -816,6 +816,23 @@ namespace Js
     {
         double result = 0;
 
+#ifdef POW_DOESNT_SUPPORT_NEGATIVE_INFINITY
+        if ( signbit( x ) && 0 == NumberUtilities::LuLoDbl( x ) && 0 == ( NumberUtilities::LuHiDbl( x ) & 0x7FFFFFFF ) )
```

Comment:
IIRC having `-0.0` here is important.

>  Either way, it would be nice to have this refactored out into its own function in NumberUtilities for readability.

This is a `pow` specific case as `POW_DOESNT_SUPPORT_NEGATIVE_INFINITY` and it's under `Math::Pow`